src/plugins/mf_priority.cpp

```diff
@@ -1131,8 +1195,19 @@ static int inactive_cb (flux_plugin_t *p,
         return 0;
 
     // this job was running, so decrement the current running jobs count
-    // and look to see if any held jobs can be released
+    // and the resources count and look to see if any held jobs can be released
     b->cur_run_jobs--;
+    if (jobspec) {
```

Comment:
under what conditions would `jobspec` be NULL? Just wondering because then a user might get stuck with the system permanently thinking they have active resources when in fact they don't.

Reply:
The only case I can think of where `jobspec` might be NULL is if for some reason the jobtap plugin failed to unpack the `jobspec` in `flux_plugin_arg_unpack ()`. Perhaps the plugin should raise an exception on the job if it fails to unpack it?

Reply:
Pretty sure that can't happen - the "pack" operation would fail and not call this.

Reply:
Oh you said unpack (sorry).  You probably want to immediately fail if the unpack fails (like due to out of memory).